LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018-Present Abaw Chen
```

Comment:
I am not sure if this should include the `graphql-python` team but looking at the other License files on this organization, it seems fine!